docs/application/dotnet/guides/applications/component-based-app.md

```diff
@@ -0,0 +1,607 @@
+# Component based applications
```

Comment:
For new files you need to update their location in https://github.com/Samsung/tizen-docs/blob/e5ed2a04bc3b759af2e7890b5dbf189c63e94b87/docs/application/toc_all.md

so that they would appear in the left nav

Reply:
I added link in ```toc_all.md```